cms/djangoapps/contentstore/views/course.py

```diff
@@ -815,6 +821,28 @@ def course_info_update_handler(request, course_key_string, provided_id=None):
     # can be either and sometimes django is rewriting one to the other:
     elif request.method in ('POST', 'PUT'):
         try:
+            if request.method == 'POST':
+                # only send bulk notifications to users when there is
+                # new update/announcement in the course.
+
+                # get the notification type.
+                notification_type = get_notification_type(u'open-edx.studio.announcements.new_announcement')
+                course = modulestore().get_course(course_key, depth=0)
+                notification_msg = NotificationMessage(
+                    msg_type=notification_type,
+                    namespace=course_key.to_deprecated_string(),
+                    payload={'course_name': course.display_name}
+                )
+
+                # get the enrolled and active user_id list for this course.
+                user_ids = CourseEnrollment.objects.values_list('user_id', flat=True).filter(
+                    is_active=1,
+                    course_id=course_key
+                )
+                if user_ids.exists():
```

Comment:
Is this a check if there are no enrollments? If you don't do this check, does something downstream fail?